### DISCUSSION TITLE
Generating PDF preview/thumbnails

### DISCUSSION THREAD

**Discussion by n1smo** [filamentphp/filament#11560] [Help]
### Package

Form builder

### Package Version

v3.2

### How can we help you?

Hi, 

im using spatie media file uploads and the spatie pdf-to-image package. 
But when im uploading a PDF there is no preview shown, even if the thumbnail gets generated.
Regular images are shown as previews.

```        
SpatieMediaLibraryFileUpload::make('file')
            ->conversion('thumb')
            ->collection('files')
```
Is this expacted behavior? Or am im missing something. And if so, is there a way to show a thumb preview of the PDF, either with spatie or the regular fileupload?

**Comment by avinmaster** [comment]
Any updates here?

**Comment by dennisvandalen** [comment]
I don't think [FilePond even supports this](https://pqina.nl/filepond/docs/api/plugins/image-preview/)
